src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,67 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private static final String COMMA = ",";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        Path pathOfCsvFile = Path.of(fromFileName);
+        try {
+            List<String> lines = Files.readAllLines(pathOfCsvFile);
+            String[] valueFromList = lines.toArray(new String[]{});
+            String report = createReport(valueFromList);
```

Comment:
Consider renaming 'valueFromList' to something more descriptive, like 'linesFromFile'.